chart/kubeapps/templates/_helpers.tpl

```diff
@@ -28,7 +28,15 @@ If release name contains chart name it will be used as a full name.
 Render image reference
 */}}
 {{- define "kubeapps.image" -}}
-{{ .registry }}/{{ .repository }}:{{ .tag }}
+{{- $image := index . 0 -}}
+{{- $global := index . 1 -}}
+{{- $registryName := $image.registry -}}
+{{- if $global }}
```

Comment:
just to double check, aren't you able to use `{{- if .Values.global -}}` here? We are using `.Values` in other places here (and you would be able to omit the changes in the other templates).

In any case, being explicit in the templates is not bad either so I am okay if we leave this like this.

Reply:
Using
```yaml
{{/*
Render image reference
*/}}
{{- define "kubeapps.image" -}}
{{- $registryName := .registry -}}
{{- if .Values.global }}
    {{- if .Values.global.regristry }}
        {{- $registryName := .Values.global.registry -}}
    {{- end -}}
{{- end -}}
{{ $registryName }}/{{ .repository }}:{{ .tag }}
{{- end -}}
```
and without changes in the other templates, tests fails because of

> Error: render error in "kubeapps/templates/tiller-proxy-deployment.yaml": template: kubeapps/templates/_helpers.tpl:32:14: executing "kubeapps.image" at <.Values.global>: can't evaluate field global in type interface {}

Reply:
👍  thanks for checking! (I don't know why that doesn't work though)

The only thing is that I think you missed the closing `-` in the if sentences:

  {{- if $global **-**}}
    {{- if $global.regristry **-**}}